### PR TITLE
[WIP] Prepare the move to Jakarta

### PR DESCRIPTION
**Pull request by gsmet**
Don't merge this one yet, there are still open questions.

/cc @emmanuelbernard

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -221,8 +221,7 @@
                     <doctitle>Jakarta Bean Validation API ${project.version}</doctitle>
                     <windowtitle>Jakarta Bean Validation API ${project.version}</windowtitle>
                     <bottom>
-                        <![CDATA[Copyright &copy; ${project.inceptionYear}-{currentYear} <a href="http://redhat.com">Red Hat Inc.</a>  All Rights Reserved.
-                        Released under the <a href="http://www.apache.org/licenses/LICENSE-2.0">Apache Software License 2.0</a>.]]>
+                        <![CDATA[This spec is licensed under the EFSL; this spec is based on material that is licensed under the Apache License, version 2.0.]]>
```
Comment (gsmet) [inline pom.xml]: @emmanuelbernard I'm not sure about this one. You said we should keep our copyright but @starksm64  asked for us to have this specific footer in the Javadoc.

File: README.md [hunk 2]
```diff
@@ -15,20 +15,20 @@ JDK 8.
...
 
 ## Contribute
 
-Want to join us? You can find all the relevant information about contributing to Bean Validation on the [website](http://beanvalidation.org/contribute/).
+Want to join us? You can find all the relevant information about contributing to Jakarta Bean Validation on the [website](https://beanvalidation.org/contribute/).
 
 ## Continuous Integration
 
-The official Continuous Integration service for the project is hosted on [ci.hibernate.org](http://ci.hibernate.org/view/Validator/).
+The official Continuous Integration service for the project is hosted on [ci.hibernate.org](https://ci.hibernate.org/view/Validator/).
```
Comment (gunnarmorling) [inline README.md]: Should likely be changed to that JIPP resource once it's there.

File: pom.xml (hunk 1 above)
Comment (gunnarmorling) [reply pom.xml]: They way I understand is that the actual sources remain under the previous (Apache) license, but that the JavaDoc is published under EFSL.

File: pom.xml [hunk 3]
```diff
@@ -8,8 +8,8 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>javax.validation</groupId>
-    <artifactId>validation-api</artifactId>
+    <groupId>jakarta.validation</groupId>
+    <artifactId>jakarta.validation-api</artifactId>
```
Comment (emmanuelbernard) [inline pom.xml]: As per @starksm64 I would keep the old artifactId: validation-api. It will be less disruptive that way.

File: pom.xml (hunk 1 above)
Comment (emmanuelbernard) [reply pom.xml]: So the license and the copyright are two different things:
- keep the copyright Red Hat

And use this text (to align to what I put in the spec)

This specification is licensed under the <a href="https://www.eclipse.org/legal/efsl.php">Eclipse Foundation Specification License 1.0</a>; this specification is based on material that is licensed under the <a href="http://www.apache.org/licenses/LICENSE-2.0">Apache Software License 2.0</a>.

Comment (emmanuelbernard) [reply pom.xml]: I am assuming that this bottom text is what is added to the javadoc and nowhere else.
Ideally, we would use a property to only activate this EFSL mention when doing a final spec release like we do swap the license-final file in the spec. But if that's too complicated let's not.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -22,3 +22,5 @@ Commits that are provided by non-committers must have a Signed-off-by field in t
 The non-committer must additionally have an Eclipse Foundation account and must have a signed Eclipse Contributor Agreement (ECA) on file.
 
 For more information, please see the [Eclipse Committer Handbook](https://www.eclipse.org/projects/handbook/#resources-commit).
+=======
```
Comment (emmanuelbernard) [inline CONTRIBUTING.md]: Why this `=======`?

**Comment by emmanuelbernard** [comment]
Done with the review @gsmet

File: pom.xml (hunk 3 above)
Comment (gsmet) [reply pom.xml]: We decided to go this way for the one we already published. All the Jakarta artifacts are `jakarta.something`.

See https://search.maven.org/search?q=g:jakarta.validation
and https://search.maven.org/search?q=g:jakarta.*

Comment (gsmet) [reply pom.xml]: BTW, right now, it uses the artifact we already published.

File: CONTRIBUTING.md (hunk 4 above)
Comment (gsmet) [reply CONTRIBUTING.md]: Dunno, fixed.

File: pom.xml (hunk 1 above)
Comment (gsmet) [reply pom.xml]: I will see what I can do once I know what we can do on Eclipse CI.

File: pom.xml (hunk 3 above)
Comment (emmanuelbernard) [reply pom.xml]: yes I saw that later. I'm making a case here https://bugs.eclipse.org/bugs/show_bug.cgi?id=548104#c31
We will see but let's not block.

**Comment by emmanuelbernard** [comment]
Note that for the spec I made a eclipse branch upstream. You might want to do the same.

**Comment by gsmet** [comment]
I pushed everything to an upstream `eclipse` branch.

**Pull request closed without merging**